doc/rfcs/001-allow-guests-to-open-multiple-remote-buffers.md

```diff
@@ -0,0 +1,47 @@
+
+Ideally, guests would be able to edit any file in the host's current project, but that will require replication of the file-system which is beyond the scope of this proposal. We can get much closer to that ideal by giving guests access to any open buffer in the host's workspace. This can be implemented without substantial changes to the current replication architecture.
+
+## Explanation
+
+### Remote buffers
+
+Just as guests can open editors for local buffers corresponding to files on their local file system, they can also open editors for *remote buffers* corresponding to buffers in any remote workspace that they have joined.
+
+### Following
+
+When you first join a portal as a guest, you're automatically *following* the host. This means that you jump to the host's current position and continue to follow them as they move between buffers.
+
+In order to jump to the host's current position, an editor is automatically added to your workspace for their active buffer when you first join their workspace. If you continue to follow the host, any time they switch to a new buffer, a new editor for that remote buffer is automatically added to your workspace and focused. Existing editors for previous remote buffers are not automatically closed when this switch occurs.
+
+If you stop following the host, either manually or automatically due to them navigating away while you are still typing, then when they change their active editor your workspace is unaffected. If you start following the host again then a new editor may be added to your workspace based on their current location so that you can continue to follow them.
+
+You can follow any other guest participating in the host's workspace in the exact same way. If they move between buffers, you will follow them. The host does not enjoy any special privilege with respect to the ability to be followed between different files.
+
+### Navigation
+
+As a guest of a shared workspace, you don't automatically receive access to the host's file system. Tools like the tree view, the file finder, and project search don't have access to their current project. (This capability will be the subject of an upcoming RFC).
+
+For now, however, the file finder *will* be augmented to make it easy for you to navigate any buffer that the host current has open in their workspace. These remote buffers will be decorated with the avatar of the host that owns their workspace.
+
+## Drawbacks
+
+Currently, a portal always corresponds to a single tab. This keeps things really simple. To leave the portal, you just close the tab. Now there could be multiple editors in your workspace that are part of a remote workspace, which increases complexity.
```

Comment:
A low-effort idea to solve this would be to add back the `Teletype: Leave Portal` command. It'd be symmetric with the `Teletype: Close Portal` command we currently offer to hosts and would be pretty easy to access too (we could even provide default keybindings for it).